gradle-guide-error-prone/src/main/java/com/palantir/gradle/guide/errorprone/taskexecution/ChainedCallMatcher.java

```diff
@@ -17,31 +17,32 @@
 package com.palantir.gradle.guide.errorprone.taskexecution;
 
 import com.google.errorprone.VisitorState;
+import com.google.errorprone.matchers.Matcher;
 import com.google.errorprone.util.ASTHelpers;
 import com.sun.source.tree.ExpressionTree;
 import com.sun.source.tree.MethodInvocationTree;
 import java.util.List;
 import java.util.Optional;
 
-public final class ChainedCall {
+public final class ChainedCallMatcher {
     /**
-     * To represent {@code foo().bar().baz()}, this list will be [baz, bar, foo].
-     * This order facilitates matching in {@code matches}
+     * To represent {@code foo().bar().baz()}, this list will be in reverse — [baz, bar, foo].
+     * This order facilitates matching in {@code ChainedCallMatcher#matches}
```

Comment:
```suggestion
     * This order facilitates matching in {@link ChainedCallMatcher#matches}
```

`@code` is for code blocks, but does not enable linking i.e. good for examples
`@link` is an actual link

Reply:
gotcha, thanks for the tip!